activities/codebase-stewardship/define-success.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
This file is in there twice, also as `success-for-a-codebase.md`.

Perhaps name this `metrics.md` and then it can be the start of our list of success metrics.

Reply:
created issue #219 & deleted duplicate